### PR TITLE
v0.79.0 — Updated menu styles.

### PR DESCRIPTION
**Pull request by DamianMullins**
### Added
- No padding collapsible card section styles.

### Changed
- Updated menu styles.

### CODE REVIEW

File: src/scss/components/_cards.scss [hunk 1]
```diff
@@ -17,8 +17,8 @@ $card-padding                                   : spacing(x2) !default;
 $card-radius                                    : 4px !default;
 $card-radius--large                             : 8px !default;
 $card-borderColor                               : $grey--lightest !default;
-$card-section-margin                            : spacing(x4) * 2 !default;
-$card-section-margin--aboveMid                  : spacing(x4) !default;
+$card-section-margin                            : spacing(x4) !default;
+$card-section-margin--aboveMid                  : spacing(x4) * 2 !default;
```
Comment (ashleynolan) [inline src/scss/components/_cards.scss]: Would it be better to add a `x8` option to the spacing map?

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,17 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
 
+v0.78.1
```
Comment (ashleynolan) [inline CHANGELOG.md]: This is a minor update 😉 Just call me the semver enforcer…

File: src/scss/components/_cards.scss [hunk 3]
```diff
@@ -92,9 +93,22 @@ $card-section-collapsible-paddingRight--aboveMid: 60px;
                 }
             }
 
+            .c-card-section--collapsible--noPad {
+                &.c-card-section {
```
Comment (ashleynolan) [inline src/scss/components/_cards.scss]: How come this needs to be included?  Is it to increase the specificity?

Comment (DamianMullins) [reply src/scss/components/_cards.scss]: It was left in after a refactor — will remove.

File: src/scss/components/_cards.scss (hunk 1 above)
Comment (DamianMullins) [reply src/scss/components/_cards.scss]: Will do, I'll as a `x7` as well, doesn't feel right to skip the lucky number! 7️⃣

**Pull request merged**